GitUI/Theming/AppColorExtension.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Drawing;
+using GitExtUtils.GitUI.Theming;
+
+namespace GitUI.Theming
+{
+    public static class AppColorExtension
+    {
+        public static Color Value(this AppColor name) =>
```

Comment:
I find callsites confusing, and not following .NET naming guidelines.
I propose to rename it as follows: 
```suggestion
        public static Color GetThemeColor(this AppColor name) =>
```

So instead of
```cs
color = AppColor.DiffAddedExtra.Value();
```
it will be
```cs
color = AppColor.DiffAddedExtra.GetThemeColor();
```

Reply:
ok